pages/desktop/details.py

```diff
@@ -18,7 +18,7 @@
 class Details(Base):
 
     # add-on information
-    _title_locator = (By.CSS_SELECTOR, 'hgroup .addon')
+    _title_locator = (By.CSS_SELECTOR, '.addon>span')
```

Comment:
Still running with this patch, but while I am: nit: whitespace between both sides of .addon and span, please (prevailing style).